cql/src/main/java/org/apache/cassandra/stargate/cql3/DefaultQueryOptions.java

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
I don't know about this one. Anything under org.apache.cassandra is probably borrowed. @mpenick ?